src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -212,10 +243,10 @@ public void Add(T item)
         /// </summary>
         public void Clear()
         {
-            if (_size > 0)
+            if (Count > 0)
             {
-                Array.Clear(_items, 0, _size);
-                _size = 0;
+                Array.Clear(_items, 0, Count); // Don't need to doc this but we clear the elements so that the gc can reclaim the references.
```

Comment:
You can remove the "Don't need to doc this but we" part and just leave it as "// clear to release any references", or something simple like that.